modules/apps/site-initializer/site-initializer-raylife/extra/direct-to-customer/package.json

```diff
@@ -18,13 +18,13 @@
 	},
 	"dependencies": {
 		"@clayui/core": "^3.32.0",
-		"@clayui/css": "3.33.1",
+		"@clayui/css": "3.35.3",
 		"@clayui/icon": "3.32.0",
 		"@craco/craco": "6.2.0",
 		"@googlemaps/js-api-loader": "^1.12.2",
 		"axios": "^0.21.1",
-		"react": "^17.0.2",
-		"react-dom": "17.0.2",
+		"react": "16.12.0",
```

Comment:
@jbalsas , I'm not sure about this 16.12.0

Last time we run SF and downgraded this version, our application stop working.

I'm not sure why, and I can test it again. But from last time this change broken our application usability.

Reply:
Even rendering everything as should, but all actions and interactivity with User and Application, such as Click of Buttons, and typing Events wasn't working properly.

Reply:
Ok, we can exclude this SF rule for now then and check later what's going on in case we want to re-enable it